response_to_reviewers.Rmd

```diff
@@ -0,0 +1,32 @@
+
+In an effort to better orient the reader to our current context, and details such as the research question from the original article, our approach to the project, and what to expect for our results, we've added a table of contents and synopsis at the top of the document.   
+
+Additionally, some reviewers had questions as to why certain parameters were added to code chunks (e.g., "echo=FALSE", which hides the code from the HTML knitted file); and so, we've added notes under the headings for each section to explain where readers can expect to find corresponding code when it is masked to produce a figure. The code for our graph includes many indentations in order to facilitate the reading of our commands in ggplot2. We have also reviewed this code to ensure that the identation remains consistent throughout. Nevertheless, we acknowledge that this approach to code writing in ggplot2 is a matter of taste.
+
+We'd like to further thank reviewers for pointing out the places wherein the comments in our code included the output statistics. We've taken better advantage of the dynamic functionality of R Markdown documents and embedded those statistical outputs into our text explanations. This will allow for any future knitted files still reflect the most-current statistics, should the underlying dataframe be amended. We've also removed some code chunk parameters so that our knitted HTML file displayed both the code, odds ratios and predicted probabilities from our logistic regression analyses. 
+
+We have also taken time to remove any "View()" functions (whether or not they were included as a comment), since this function can interfere with furture knitting of the RMD file. Many of our variable names have also been amended to feature a consistent coding style throughout the document (e.g., all variable names are no longer capitalised). This will also help the reader better distinguish between when we are assigning/recoding variables and when we are configuring them as labels for our figures in the code. 
+
+Although we acknowledge that some variable names are not entirely descriptive, we've kept many of the names in order to be conscise in our coding style. However, where necessary, we have included more descriptive comments to better help guide our readers. Along those same lines, in our comments we've attempted to better describe the places in our code where we are merely checking our assumptions ahead of statistical analyses and where we are performing tests to directly compare our output to that from the original article. Also, some of the variable naming, particularly within functions, e.g.: "pwr.chisq.test()", was specific to the command structure of our analyses; and so we could not change them. However, we do strongly encourage the use of the R-helper shortcuts (i.e., "??[function here]") for further review of the command structures for functions, should our readers have questions about what certain abbreviations stand to represent. 
```

Comment:
Why not value clarity over conciseness?  In my opinion clarity is more valuable.  Remember how hard it was for you to work with the variables without a code book?  I think you want to try to write your code in such a way that you can avoid that in the future.